src/source/Ice/IceAgent.c

```diff
@@ -354,6 +355,7 @@ STATUS iceAgentAddRemoteCandidate(PIceAgent pIceAgent, PCHAR pIceCandidateString
                 STRTOUI32(curr, next, 10, &priority);
                 break;
             case SDP_ICE_CANDIDATE_PARSER_STATE_PROTOCOL:
+                STRNCPY(remoteProtocol, curr, tokenLen);
```

Comment:
string is not null terminated, but below is appears that 'protocol' is meant to be a null terminated string.
Please add remoteProtocol[tokenLen] = '\0';

Reply:
Also, need to add a check that tokenLen < MAX_PROTOCOL_LENGTH or we have an overwrite security risk

Reply:
`\0` initialized the string instead.